front/src/modules/simulationResult/components/SimulationResultsMap/RenderItinerary.tsx

```diff
@@ -11,14 +13,36 @@ interface RenderItineraryProps {
 export default function RenderItinerary(props: RenderItineraryProps) {
   const { geojsonPath, layerOrder } = props;
 
-  const paint = {
-    'line-width': 3,
-    'line-color': '#82be00',
+  const paintBackgroundLine = {
+    'line-width': 4,
+    'line-color': '#EDF9FF',
   };
 
+  const paintLine = {
+    'line-width': 1,
+    'line-color': '#158DCF',
+  };
+
+  const startCoordinate = geojsonPath.geometry.coordinates[0];
+  const endCoordinate =
+    geojsonPath.geometry.coordinates[geojsonPath.geometry.coordinates.length - 1];
```

Comment:
- use .at() instead of `[...]`
- you need to handle the case where the coodinates are undefined
```suggestion
  const startCoordinate = geojsonPath.geometry.coordinates.at(0);
  const endCoordinate =
    geojsonPath.geometry.coordinates.at(- 1);
```

Reply:
Done

Reply:
Seems good to me to have them on the wrapped map as well.
Anyway, I think we will have to think about this wrapped map globally so we can adjust it then. Furthermore, since the map is backwards compared to the Y-axis of the time-space chart, the departure and arrival icons limit the confusion risk.